Use printf instead of echo for some shells.

http://www.cygwin.com/ml/libc-alpha/2005-02/msg00123.html

This patch does not generate invalid csu/version-info.h which has
unterminating " line because some /bin/sh cannot parse correctly.  The
previous discussion was:

	http://sources.redhat.com/ml/libc-alpha/2004-08/msg00129.html
	http://sources.redhat.com/ml/libc-alpha/2004-08/msg00145.html

This patch gets rid of all this issue because the conclusion of the
discussion was to use printf instead of echo.  Roland, could you look
at it again?

Regards,
-- gotom

2005-02-26  GOTO Masanori  <gotom@debian.or.jp>

	* csu/Makefile: Use printf instead of echo for some shells.

---
 csu/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: glibc-2.3.6/csu/Makefile
===================================================================
--- glibc-2.3.6.orig/csu/Makefile
+++ glibc-2.3.6/csu/Makefile
@@ -241,7 +241,7 @@ $(objpfx)version-info.h: $(common-objpfx
 	 esac; \
 	 files="$(all-Banner-files)";				\
 	 if test -n "$$files"; then				\
-	   echo "\"Available extensions:\\n\"";			\
+	   printf '"Available extensions:\\n"\n';		\
 	   sed -e '/^#/d' -e 's/^[[:space:]]*/	/'		\
 	       -e 's/^\(.*\)$$/\"\1\\n\"/' $$files;		\
 	 fi) > $@T
